bot/plugins/help.py

```diff
@@ -0,0 +1,57 @@
+import crescent
+import hikari
+
+from bot.app import Plugin
+from bot.constants import EMBED_COLOR
+
+plugin = Plugin()
+
+HELP_EMBEDS = [
+    hikari.Embed(
+        description=(
+            "Hi! My name is io, and my job is to run code."
+            "\n I can run any code inside of code blocks:"
+            "\n\\`\\`\\`<language-name>"
+            "\n<your code here>"
+            "\n\\`\\`\\`"
+        ),
+        color=EMBED_COLOR,
+    ),
+    hikari.Embed(
+        description=(
+            "\n\\* Running code - Start your message with `io/run`."
+            "\n\\* View Assembly - Start your message with `io/asm`."
```

Comment:
Why not use discord's list markdown? Start each line with `-` (no space before)

Reply:
i switched to discord's markdown list